notebooks/0-the-cloud/0-the-cloud.ipynb

```diff
@@ -0,0 +1,653 @@
+{
```

Comment:
<p>I think we should encourage users to use the query_object or query_criteria functions. We don't want people copy-pasting coordinates from simbad since that's where we get object coordinates from anyway</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/project-tikebook/pull/7/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='44'/>

Reply:
<p>Makes sense, changed text</p><p> </p>